### ISSUE TITLE
no instructions

### ISSUE THREAD

**Issue by vchunkyzv** [Aeryle/library-gui#6]
I'm having problems running this. I downloaded the zip and extracted but I cannot see any "app" to run to open the GUI, any other file is json or doesn't open. The instructions don't really say what to do it just says what it is. I found an index html page but that opens blank for me when I click on the options there. What am I missing? any help would be greatly appreciated thanks.

**Comment by Aeryle** [comment]
You are supposed to directly download the exe in the releases page: https://github.com/Aeryle/library-gui/issues
Or if you are a developer, or just want the whole latest files, follow the instructions in the README.md file.

**Comment by vchunkyzv** [comment]
I guess that's my issue. I didn't get an exe when I downloaded the zip
file. I'm far from a developer which is why I'm having issues with this. I
just want to change the purple color on the library I just modified. I know
a bit of xml and html and can usually fend for myself but I'm stomped with
this atm. I'm sure is something simple but still trying to change it.
thanks for replyinig btw. Where is the exe supposed to be?
